Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -103,5 +103,11 @@ void Highlight::repaint()
         repaintRange(data->range());
 }
 
+void Highlight::setPriority(int priority)
+{
+    if (priority >= 0)
```

Comment:
Is it really valid to exclude negative numbers?

The [spec](https://webidl.spec.whatwg.org/#idl-long) defines the attribute as:

```
attribute long priority;
```

Note the lack of `unsigned`.

Reply:
You're right, just found "Left the priority attribute as (implicitly) 'signed'." Will change to support negative integers.